vendor/github.com/go-openapi/strfmt/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,35 @@
+// +build !ignore_autogenerated
```

Comment:
I'm open for suggestions where should I add this file. Or for general suggestions how we should add DeepCopy method to strfmt.

I was thinking about having a fork of strfmt, but that might be too huge effort for just one file.

Reply:
Can we submit it upstream?

Reply:
Good idea, done https://github.com/go-openapi/strfmt/pull/44

Reply:
Upstream PR merged. I'm going to bump go-openapi version and update the PR.

Reply:
Awesome!